sql/2023/01/critical-css-opportunity.sql

```diff
@@ -26,7 +26,8 @@ FROM (
     pages._TABLE_SUFFIX AS client,
     COUNT(pages.url) AS total_wp_sites,
     COUNTIF( CAST(JSON_EXTRACT_SCALAR(payload, '$._renderBlockingCSS') AS INT64) = 0
-      AND CAST(JSON_EXTRACT_SCALAR(payload, '$._inline_style_bytes') AS INT64) > 0 ) AS sites_with_critical_css
+      AND CAST(JSON_EXTRACT_SCALAR(payload, '$._inlineCssInHead') AS INT64) > 0
+      AND ( CAST(JSON_EXTRACT_SCALAR(payload, '$._externalCssInHead') AS INT64) + CAST(JSON_EXTRACT_SCALAR(payload, '$._externalCssinBody') AS INT64) ) > 0 ) AS sites_with_critical_css
```

Comment:
Similar feedback here as in my other comment above, however as mentioned above, please put this into a new query.

Here is what that entire `COUNTIF` condition needs to look like for this one:
```sql
    COUNTIF(
      CAST(JSON_EXTRACT_SCALAR(payload, '$._renderBlockingCSS') AS INT64) = 0
      AND CAST(JSON_EXTRACT_SCALAR(JSON_EXTRACT_SCALAR(payload, '$._css'), '$.inlineCssInHead') AS INT64) > 0
      AND CAST(JSON_EXTRACT_SCALAR(JSON_EXTRACT_SCALAR(payload, '$._css'), '$.externalCssInBody') AS INT64) > 0
    ) AS sites_with_critical_css
```

Reply:
Update with code in the comment.